test/unit/models/classes/Security/AuthorizationServerFactoryTest.php

```diff
@@ -42,8 +42,8 @@ class AuthorizationServerFactoryTest extends TestCase
 {
     use NoPrivacyTrait;
 
-    private const PRIVATE_KEY_PATH = __DIR__ . '/../../../../../vendor/oat-sa/lib-lti1p3-core/tests/Resource/Key/RSA/private.key';
-    private const PUBLIC_KEY_PATH = __DIR__ . '/../../../../../vendor/oat-sa/lib-lti1p3-core/tests/Resource/Key/RSA/public.key';
+    private const PRIVATE_KEY_PATH = __DIR__ . '/../../../../../../vendor/oat-sa/lib-lti1p3-core/tests/Resource/Key/RSA/private.key';
+    private const PUBLIC_KEY_PATH = __DIR__ . '/../../../../../../vendor/oat-sa/lib-lti1p3-core/tests/Resource/Key/RSA/public.key';
```

Comment:
This causes the CI pipeline error.
```
1) unit\models\classes\Security\AuthorizationServerFactoryTest::testCreate

file_get_contents(/var/www/test/unit/models/classes/Security/../../../../../../vendor/oat-sa/lib-lti1p3-core/tests/Resource/Key/RSA/private.key): failed to open stream: No such file or directory
```
Let's put it back :)
```suggestion
    private const PRIVATE_KEY_PATH = __DIR__ . '/../../../../../vendor/oat-sa/lib-lti1p3-core/tests/Resource/Key/RSA/private.key';
    private const PUBLIC_KEY_PATH = __DIR__ . '/../../../../../vendor/oat-sa/lib-lti1p3-core/tests/Resource/Key/RSA/public.key';
```